pkgs/by-name/du/duckstation/package.nix

```diff
@@ -57,26 +63,27 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     SDL2
     curl
+    extra-cmake-modules
     libXrandr
     libbacktrace
     qtbase
     qtsvg
     qtwayland
     wayland
-  ]
-  ++ cubeb.passthru.backendLibs;
+  ] ++ cubeb.passthru.backendLibs;
+
+  cmakeFlags = [ (lib.cmakeBool "BUILD_TESTS" true) ];
```

Comment:
This attribute also switched places, and you removed the line break and indentation, with no justification.

Reply:
The "justification" is nixfmt.